app/src/main/kotlin/com/simplemobiletools/calculator/helpers/MoneyCalculatorImpl.kt

```diff
@@ -98,14 +98,14 @@ class MoneyCalculatorImpl(calculator: Calculator, taxCalculator: TaxCalculator)
     }
 
     fun performTaxing(location:String){
-        overwriteNumber(TaxOperation(getResult(), location).getResult())
+        overwriteNumber(TaxOperation.setParams(getResult(), location)!!.getResult())
```

Comment:
Maybe we can abstract this bit if code: 

`_TaxOperation.setParams(getResult(), location)!!.getResult()_`

It's used for all the 3 monetary features.

Reply:
The signature is different, TaxOperation.setParams(double, string), TipOperation.setParams(double, double), which prevents the abstraction.

Reply:
Okay that makes sense. Leave it as is then